src/api/participantsRouter.ts

```diff
@@ -49,6 +53,23 @@ export function createParticipantsRouter() {
     return res.status(200).json(participants.map(mapParticipantToAvailableParticipant));
   });
 
+  participantsRouter.get('/awaitingApproval', async (req: ParticipantRequest, res) => {
```

Comment:
This logic is a little messy, so thinking of either trying to simplify it or at least moving to the service? Let me know what you think

Reply:
I like the idea of moving it into a service.

Reply:
done!